### ISSUE TITLE
Support for IE8 & browser prefixes?

### ISSUE THREAD

**Issue by joelpittet** [ubc-web-services/Megatron#36]
Wondering if you are supporting IE8. if we don't need to then maybe we can remove the browser prefixes?

`box-shadow` for example, won't work in IE8 and the prefixes were removed in FF3, Opera 10, Chrome 9(maybe)

http://caniuse.com/#search=box-shadow

**Comment by occupant** [comment]
We're only supporting IE9+ (web services) but the CLF as distributed by Web Communications does (they support IE7 even, poor souls). I assume you're referring to the includes > config.php file and the unit-custom less and css files? I was initially trying to follow their lead in terms of exhaustive support, but that's not really defensible these days.

I've been thinking those should come out and I think the conditional IE browser classes in html-tpl.php have seen their day as well. And it would be good to reroll modernizer to slim it down.The clf_drupal.css file is in need of a good prune too (it's got several questionable additions and some things that are redundant).

**Comment by joelpittet** [comment]
Yeah I guess it would be a bit tricky if we are using their style from the CDN because you have to override all the prefixes used upstream eh?  How about taking a local copy of the CLF and running it through auto-prefixer in like a grunt/gulp task?

**Comment by occupant** [comment]
We can do that (and I have a similar repo for this here: https://github.com/ubc-web-services/ubc-clf-min). The only real issue is that the Web Communications team prefers the assets used from the CDN. Their rationale is that it will be cached between sites and usage can be tracked (which is handy considering we don't have a university-wide GA account).

That said, there's nothing really preventing pulling the assets, modifying them to suit, and serving them locally. The only thing that should absolutely come from the CDN is the central js file - it's what they use to update the header / footer content and links as well as rewrite the UBC100 region when changing campaigns.

I'm working with them now to see what we can do about creating an 'exploded view' CLF so you can pick and choose your includes, and see about having a leaner version all hosted on the central CDN. That's still at least a few months out though.
